Sources/ProjectDescription/Template/AttributeHelpers.swift

```diff
@@ -0,0 +1,36 @@
+import Foundation
+
+/// Error that occurs when parsing attributes from input
+enum ParsingError: Error, CustomStringConvertible, Equatable {
+    /// Thrown when could not find attribute in input
+    case attributeNotFound(String)
+    /// Thrown when attributes not provided
+    case attributesNotProvided
+
+    var description: String {
+        switch self {
+        case let .attributeNotFound(name):
+            return "Could not find attribute \(name)"
+        case .attributesNotProvided:
+            return "You must provide --attributes option"
+        }
+    }
+}
+
+/// Returns value for `ParsedAttribute` of `name` from user input
+/// - Parameters:
+///     - name: Name of `ParsedAttribute`
+///     - arguments: Arguments to get attribute from (usually you should lead this at default)
+/// - Returns: Value of `ParsedAttribute`
+public func getAttribute(for name: String, arguments: [String] = CommandLine.arguments) throws -> String {
```

Comment:
As I mentioned in the other PR, the models in `ProjectDescription` model should be dumb and the logic to substitute the variables should be outside the `ProjectDescription` framework.